src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -1158,8 +1158,11 @@ public void run()
         }
         catch (SocketException ex)
         {
-            if (!"Socket closed".equals(ex.getMessage()))
+            if (!"Socket closed".equals(ex.getMessage())
```

Comment:
But we are only catching SocketException-s in this block. Am I missing something?